app/services/repository_fetcher.rb

```diff
@@ -0,0 +1,28 @@
+class RepositoryFetcher
+  GITHUB_BASE_URL = 'https://api.github.com'
```

Comment:
Noticed in this https://robots.thoughtbot.com/faking-apis-in-development-and-staging post, that they freeze the BASE_URL string -- imagine so that can ensure it will never get modified and thus impact its use later in a `get`.